pkg/collect/copy.go

```diff
@@ -109,9 +115,59 @@ func copyFiles(c *Collector, client *kubernetes.Clientset, pod corev1.Pod, copyC
 		return nil, errors
 	}
 
-	return map[string][]byte{
-		copyCollector.ContainerPath: output.Bytes(),
-	}, nil
+	for _, file := range filepaths {
+		if file != "" {
```

Comment:
This would be better written has 

```
if file == "" {
  continue
}
```

The code is less conditionally indented and more readable this way

Reply:
@CodeLingoBot capture continue in conditional in loop